src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -298,6 +297,9 @@ CreateConceptForm.defaultProps = {
   isEditConcept: false,
   handleAsyncSelectChange: () => {},
   handleAnswerChange: () => {},
+  // eslint-disable-next-line react/default-props-match-prop-types
```

Comment:
Is this line necessary for the commit or just a local setting to your linter?

Reply:
Fixed